### PR TITLE
Apprentice action fix

### PR DESCRIPTION
**Pull request by djtjwillia**

### CODE REVIEW

**Review by austinjw — DISMISSED** [review]

File: apprentice-action/tests/endpoint.test.js [hunk 1]
```diff
@@ -3,27 +3,27 @@ const expect = require("chai").expect;
 
 describe("Tests to the \"/\" endpoint", () => {
     it("should return a 200 status code", async () => {
-        const res = await axios("http://host.docker.internal:80/");
+        const res = await axios("http://172.17.0.1:80/");
```
Comment (mrparkers) [inline apprentice-action/tests/endpoint.test.js:6]: did you mean to hardcode this?

Comment (djtjwillia) [reply apprentice-action/tests/endpoint.test.js:6]: yes, we found that the internal ip worked, while the dns name did not.

Comment (austinjw) [reply apprentice-action/tests/endpoint.test.js:6]: https://stackoverflow.com/questions/65497331/github-actions-is-it-possible-to-call-the-runner-host-from-a-docker-container

Comment (chodges7) [reply apprentice-action/tests/endpoint.test.js:6]: It doesn't look like we can resolve this URL unless we add a [host in at runtime](https://medium.com/@TimvanBaarsen/how-to-connect-to-the-docker-host-from-inside-a-docker-container-112b4c71bc66). Which looks difficult to implement

**Review by austinjw — APPROVED** [review]

**Pull request merged**